packages/build/src/steps/run_core_steps.js

```diff
@@ -186,9 +191,10 @@ const runBuildStep = async function ({
     debug,
     timers: [],
     featureFlags,
+    // eslint-disable-next-line max-lines
```

Comment:
We typically add an exception to this rule at the top of the file (i.e. `/* eslint-disable max-lines */`), or even an exception in https://github.com/netlify/build/blob/main/.eslintrc.cjs.

Reply:
I added the exception in the eslint config now :)